### DISCUSSION TITLE
Timing isse v2 -> v3

### DISCUSSION THREAD

**Discussion by ChrisSantiago82** [livewire/livewire#8102] [1. Help]
I have an app that I'm upgrading from v2 to v3. Run into an issue. In v2 it works fine and in V3 it breaks.

It's a modal with a form and model binding (yes, I know modal binding is legacy).

It's a form/modal I'm reusing for creating or editing models. At the beginning of the blade I have "@if ($modal !== null)". So the inputs would only be shown if there is a model. On the input I have something like "wire:model="model.name" for the binding.

When the model is being saved, by pressing a button, I set the model to null, hence the inputs should disappear. But then I'm getting an error "Uncaught TypeError: Cannot read properties of null (reading 'name')".

Livewire is still trying to do the wire:model="model.name" binding, but actually, it shouldn't exist anymore because before that I have the "@if ($modal !== null)".

Did anybody has similar issues?

**Comment by joshhanley** [comment]
@ChrisSantiago82 any chance you could create a `https://wirebox.app` or a failing test PR that demonstrates the issue so we can have a look? Thanks!
